api/v1alpha2/project_types.go

```diff
@@ -167,9 +193,7 @@ type ProjectStatus struct {
 //+kubebuilder:printcolumn:name="Project Name",type=string,JSONPath=`.status.name`
 //+kubebuilder:printcolumn:name="Project ID",type=string,JSONPath=`.status.id`
 
-// Project manages HCP Terraform Projects.
-// More information:
-//   - https://developer.hashicorp.com/terraform/cloud-docs/projects/manage
+// Project is the Schema for the projects API
```

Comment:
Could you please revert this change? :) We need this for Red Hat OpenShift for better documentation.